index.js

```diff
@@ -1,13 +1,16 @@
 /* eslint-disable max-classes-per-file */
 
 // Book Function
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
 // Store Class: Handles local storage of books
-class Storage {/* eslint-disable max-classes-per-file */
+class Storage {
+  /* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Great job working on this milestone. You are not suppose to disable eslint, rather fix the errors resulting from running the linter checks.